pkg/volume/gce_pd/gce_pd.go

```diff
@@ -56,6 +58,18 @@ const (
 	gcePersistentDiskPluginName = "kubernetes.io/gce-pd"
 )
 
+// The constants and DiskAttachLimit are used to map from the machine type (number of CPUs) to the limit of
+// persistant disks that can be attached to an instance. Please refer to gcloud doc
+// https://cloud.google.com/compute/docs/disks/#increased_persistent_disk_limits
+const (
+	Default = iota
+	OneCPU
+	TwotoFourCPUs
+	EightCPUsandAbove
+	EightCPUs = 8
```

Comment:
I still think it would be better to separate this out into 2 different lists.  It's a bit confusing that it's mixing enum with non-enum usage.

Reply:
I changed the list to enum. PTAL. Thanks!